app/cdap/components/Snackbar/styles.ts

```diff
@@ -13,33 +13,34 @@
  * License for the specific language governing permissions and limitations under
  * the License.
  */
+
 import { makeStyles } from '@material-ui/core';
-import { blue } from '@material-ui/core/colors';
-import { grey } from '@material-ui/core/colors';
+import { green } from '@material-ui/core/colors';
+import { red } from '@material-ui/core/colors';
```

Comment:
we can add both imports to single statement

Reply:
Done